src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -643,4 +644,36 @@ public static String getVerificationString(Guild.VerificationLevel level) {
                 return level.toString().charAt(0) + level.toString().substring(1).toLowerCase();
         }
     }
+
+    public static Long parseTime(String time) {
+        List<String> toReverse = Arrays.asList(time.split(":"));
+        Collections.reverse(toReverse);
+        String timeToUse = toReverse.stream().collect(Collectors.joining(":"));
+        Matcher digitMatcher = Pattern.compile("^([0-9]*):?([0-9]*)?:?([0-9]*)?$").matcher(timeToUse);
+        if (digitMatcher.matches()) {
+            int seconds = GeneralUtils.getInt(digitMatcher.group(1), -1);
+            int minutes = GeneralUtils.getInt(digitMatcher.group(2), -1);
+            int hours = GeneralUtils.getInt(digitMatcher.group(3), -1);
+            if (hours != -1 && minutes != -1 && seconds != -1) {
+                return (long) (((hours * 60 * 60) + (minutes * 60) + seconds) * 1000);
+            } else if (minutes != -1 && seconds != -1) {
+                return (long) (((minutes * 60) + seconds) * 1000);
+            } else if (seconds != -1) {
+                return (long) (seconds * 1000);
```

Comment:
You know this could probably be handled a lot better, we could actually just stick this into Joda Time.

Reply:
How?